### DISCUSSION TITLE
homebrew Formula

### DISCUSSION THREAD

**Discussion by andre68723** [topgrade-rs/topgrade#46] [General]
Hi guys, 

almost missed the fork/discontinued topgrade. Do you plan to update the Formula or create a new one?
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/topgrade.rb

Thanks!

**Comment by s34m** [comment]
This is referenced in Issue #42 and I'm working on the change. I hope they allow the change, else I have to make a new Formula for topgrade-rs.

**Comment by s34m** [comment]
This repo is now the repo in the Formula
